glom/core.py

```diff
@@ -1199,19 +1242,19 @@ def _t_eval(target, _t, scope):
             try:
                 cur = getattr(cur, arg)
             except AttributeError as e:
-                raise PathAccessError(e, Path(_t), i // 2)
+                raise PathAccessError(e, Path(_t), i // 2) from None
```

Comment:
I don't think this syntax is py2 compatible, but I take it you want to clear out the chained exception?

Reply:
fixed this, I think